skins/oasis/css/core/PageHeader.scss

```diff
@@ -3,40 +3,44 @@
 @import 'skins/shared/mixins/flexbox';
 @import 'layout';
 
-.WikiaPageHeader {
-	margin: 0 10px 10px;
-	position: relative;
-
-	&.separator {
-		border-bottom: 1px solid $color-page-border;
-		padding: 0 0 15px;
+.page-header {
```

Comment:
:question: has this been addressed in 829ea998b9369645ba8b2bc5aa96f91c882a8e5e ?